ui/imports/shared/views/HistoryView.qml

```diff
@@ -20,15 +20,15 @@ import "../controls"
 ColumnLayout {
     id: root
 
-    property var account
+    property var assets
```

Comment:
This will change in future PR. This PR in an intermediate step to init the overview module. Later on, all module will depends on a combined filter: addresses/chain ids
This is why I remove the concept of "current" or "account"